mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -196,10 +196,32 @@ private function getEmailVolumeLimitReachedMessage(): string {
     return "{$message}<br/>";
   }
 
+  private function getPendingApprovalMessage(): string {
+    $message = __("Your subscription is currently [link]pending approval[/link].You'll soon be able to send once our team reviews your account. In the meantime, you can send previews to your authorized emails.", 'mailpoet');
```

Comment:
If I remember correctly, we prefer typographic apostrophes in English texts. It means `’` instead of `'`. Please, could you replace it here?